deployment/helmfile.yaml

```
@@ -31,19 +31,23 @@ releases:
             environment: {{ env "ENVIRONMENT" | default .Environment.Name }}
             glitchTipDsn: {{ env "GLITCH_TIP_DSN" }}
             zones:
-              kubernetesVersion:
-                textColor: 'white'
-                backgroundColor: '#3371e3'
-                label: 'Kubernetes'
-              openshiftVersion:
-                textColor: 'white'
-                backgroundColor: '#EE0000'
-                label: 'OpenShift'
-              sdnPlugin:
-                textColor: '#0b3046'
-                backgroundColor: '#ffd902'
-                label: 'SDN Plugin'
-              memoryCPUratio:
-                label: 'Memory CPU Ratio'
-                textColor: 'white'
-                backgroundColor: '#0b3046'
+              zoneFeatures:
+                kubernetesVersion:
+                  textColor: 'white'
+                  backgroundColor: '#3371e3'
+                  label: 'Kubernetes'
+                openshiftVersion:
+                  textColor: 'white'
+                  backgroundColor: '#EE0000'
+                  label: 'OpenShift'
+                sdnPlugin:
+                  textColor: '#0b3046'
+                  backgroundColor: '#ffd902'
+                  label: 'SDN Plugin'
+                memoryCPUratio:
+                  label: 'Memory CPU Ratio'
+                  textColor: 'white'
+                  backgroundColor: '#0b3046'
+              zoneURLLabels:
+                kubernetes: Kubernetes API
```

```suggestion
                kubernetesAPI: Kubernetes API
```
Minor nitpick: I think for the preview to not show `Kubernetesapi` the label should be `kubernetesAPI`